static/js/query-configs/websites.ts

```diff
@@ -0,0 +1,20 @@
+import { Website } from "../types/websites"
+
+interface Websites {
+  string: Website
+}
```

Comment:
I believe this type as written is typing an object shaped like this:

```ts
{ "string": { ... } }
```

I think we want to do this instead:

```ts
interface Websites {
  [key: string]: Website
}
```

Reply:
Ah good catch